QuizApp/Quiz/QuizListViews/QuizListView.swift

```diff
@@ -76,6 +86,15 @@ struct QuizListView: View {
         }
     }
 
+    private func setErrorView(_ error: QuizError) -> (String, String) {
```

Comment:
This does not actualy set error view. Consider renaming it to something more like resolve error message. Also, it might be a better solution to put this logic into ViewModel and have both `serverError` and `empty` have an associated value of type 
```swift
struct ErrorInfo {
    
    let title: String
    let description: String

}
```
So you can just pass around an enum, rather than an enum and then resolving properties somewhere else.